### PR TITLE
[Orca-111]/secrets management

### PR DESCRIPTION
**Pull request by BWMac**
This PR adds the AWS Secrets Manager backend to our base airflow implementation. by adding `config/airflow.cfg `and mounting it in `docker-compose.yml` we can customize the configuration of airflow and connect it to AWS Secrets Manager.  The role that is already assigned to our ec2 instance was updated to be able to access secrets stored in dnt-dev and the secrets were updated to provide the access token when retrieved with `Variable.get()`.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 apache-airflow = "*"
 sagetasks = "*"
+boto3 = "*"
```
Comment (BrunoGrandePhD) [inline Pipfile:9]: Do you also need to install the AWS provider package?

**Review by BrunoGrandePhD — COMMENTED** [review]
Overall, this looks good. I just had one question before approving.

File: Pipfile (hunk 1 above)
Comment (BWMac) [reply Pipfile:9]: the aws provider package comes with `apache-airflow`, it shows up in `Pipfile.lock `and `requirements.txt`, but not in `Pipfile`

Comment (BrunoGrandePhD) [reply Pipfile:9]: I should have just checked the `Pipfile.lock` file. My bad!

**Review by BrunoGrandePhD — APPROVED** [review]

**Pull request merged**